src/lib/minions/data/killableMonsters/custom/bosses/Naxxus.ts

```diff
@@ -0,0 +1,101 @@
+import { Time } from 'e';
+import LootTable from 'oldschooljs/dist/structures/LootTable';
+
+import { naxxusCL } from '../../../../../data/CollectionsExport';
+import { GearStat } from '../../../../../gear';
+import { KillableMonster } from '../../../../../minions/types';
+import { makeKillTable } from '../../../../../util/setCustomMonster';
+
+const runes = new LootTable()
```

Comment:
some measurements/simulations on the loot would be good, like how much per hour, maybe how much loot you'd get finishing the cl